engine/src/main/resources/assets/i18n/menu.lang

```diff
@@ -128,6 +128,7 @@
     "existing-file-warning": "existing-file-warning",
     "exit-game": "exit-game",
     "extra-lighting": "extra-lighting",
+    "extra-menu-title": "extra-menu-title",
```

Comment:
Since this menu is effectively the same (although the contents vary slightly) as the existing extras menu you can just reuse its string instead - we don't need two translation strings for the same word :-)